src/hooks/listbox/merge-listbox-options.ts

```diff
@@ -0,0 +1,33 @@
+import { IListBoxOptions, IListLayout } from "../types";
+import { postProcessPages } from "./funcs";
+
+export default function mergeListboxOptions(listboxOptions: IListBoxOptions, fieldName: string, layout: IListLayout) {
+  const listBoxOptionsDefaults: IListBoxOptions = {
+    dqEnabled: false,
+    update: () => { },
+  };
+
+  const o = {
+    ...listBoxOptionsDefaults,
+    ...listboxOptions || {},
+  };
+
+  return {
+    title: fieldName,
+    search: o.search, // && (!dqEnabled || (dqEnabled && isEnabled('DIRECT_QUERY_SEARCH'))),
+    toolbar: false,
+    checkboxes: o.dqEnabled && o.checkboxes !== false,
```

Comment:
Don't need to hard-code `dqEnabled` (sense-client specific code). This logic should be on the sense-client side for that use-case.

Reply:
We should probably let the user set these options as s/he wishes without any sense-client bias.